lib/CamDiag.py

```diff
@@ -81,7 +81,9 @@
     #Add all sub-directories to python path:
     for dirname in dirs:
         sys.path.append(os.path.join(root,dirname))
-
+print(100*"=")
```

Comment:
Can you comment these `print` statements out and add an extra comment stating that they need to be added to a debug log?